Migrate BlueConfig keys to SONATA simulation config (#8)
BlueBrain/sonata-extension: source/sonata_simulation.rst

   spike_location                  text       Optional    The spike detection location. Can be either ‘soma’ or 'AIS' for detecting spikes in either the soma or axon initial segment, respectively. Default is 'soma'.
   integration_method              text       Optional    Selects the NEURON/CoreNEURON `integration method <https://nrn.readthedocs.io/en/latest/python/simctrl/programmatic.html#secondorder>`_. This parameter sets the NEURON global variable h.secondorder. The allowed values are ‘0’ (default) for fully implicit backward euler, ‘1’ for Crank-Nicolson and ‘2’ for Crank-Nicolson with fixed ion currents.
   stimulus_seed                   integer    Optional    A non-negative integer used for seeding noise stimuli and any other future stochastic stimuli, default is 0.
   ionchannel_seed                 integer    Optional    A non-negative integer used for seeding stochastic ion channels, default is 0.

>    minis_seed                      integer    Optional    A non-negative integer used for seeding the Poisson processes that drives the minis, default is 0.

Reviewer: ```suggestion
   minis_seed                      integer    Optional    A non-negative integer used for seeding the Poisson processes driving the minis, default is 0.
```

   Property                       Type       Requirement  Description
   ============================== ========== ============ ==========================================
   voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.

>    rs                             float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.

Reviewer 2: would it make sense to call this `resistance` or `series_resistance`?

Reviewer: `series_resistance` would definitely be more human-readable

   Property                       Type       Requirement  Description
   ============================== ========== ============ ==========================================
   voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.

>    rs                             float      Optional     Specifies the series resistance in Mohm. Default is 0.01 Mohm.

Reviewer 3: ```suggestion
   rs                             float      Optional     Specifies the series resistance in MΩ. Default is 0.01 MΩ.
```

   Property                       Type       Requirement  Description
   ============================== ========== ============ ==========================================
   voltage                        float      Mandatory    Specifies the membrane voltage the targeted cells should be held at in mV.

>    rs                             float      Optional     Specifies the series_resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.

Reviewer: ```suggestion
   series_resistance              float      Optional     Specifies the series resistance in M :math:`\Omega`. Default is 0.01 M :math:`\Omega`.
```

Author: Yes, the last change that I've pushed has this change in.
